src/env/schema.mjs

```diff
@@ -64,7 +64,8 @@ export const serverSchema = z.object({
     const str = String(value);
     return str.split(',');
   }, z.array(z.string())),
-  IMAGE_INGESTION_MESSAGE_QUEUE_SERVER: z.string().optional()
+  IMAGE_INGESTION_MESSAGE_QUEUE_SERVER: z.string().optional(),
+  RPC_TIMEOUT: z.number().optional(),
```

Comment:
I get a error when specify RPC_TIMEOUT=10000 in .env zod is asking for a number but somehow .env variable is read as string

Reply:
Updated code in main should fix this.